### ISSUE TITLE
Error Filling X1C

### ISSUE THREAD

**Issue by gvogele** [bambulab/BambuStudio#2954]
### Bambu Studio Version

Last

### Where is the application from?

Bambu Lab Official website

### OS version

W11

### Printer

X1C

### How to reproduce

the Print fills Gaps with Fillings. There is no Filling to do.

### Actual results

Part not usable

### Expected results

No Fillings

### Project file & Debug log uploads

[shower adapter.zip](https://github.com/bambulab/BambuStudio/files/13462227/shower.adapter.zip)

### Checklist of files to include

- [ ] Log file
- [X] Project file

**Comment by gvogele** [comment]
With Tree Mode it works, ??

**Comment by gvogele** [comment]
Yes Correct it shows everyting correct but if i Print it will make fills

**Comment by QingZhangBambu** [comment]
could you upload your gcode.3mf? The concerned staff tends to reproudce it.

**Comment by gaolegao-lx** [comment]
**Please upload log through Handy APP, and give us your ticket number**
***
***Upload log files through Bambu Handy***
     Log files from the Bambu printer can be uploaded via Bambu Handy over the network, with an upload speed of around 150 kbps under optimal network conditions.
     When uploading logs, you have the option to select a specific time frame, allowing you to reduce the volume of logs transmitted and shorten the upload duration.
     To upload files, go to **Me** → **Support Tickets** → Choose the ticket for log upload and proceed with the operation.
     ![app_support_ticket](https://wiki.bambulab.com/p1/troubleshooting/upload-log/app_support_ticket.png)
